internal/pkg/cli/errors.go

```diff
@@ -163,3 +163,25 @@ func (e *errPipelineDependsOnEnv) RecommendActions() string {
 or run %s to delete the pipeline before running %s to delete the environment`,
 		e.pipeline, e.env, color.HighlightCode(fmt.Sprintf("copilot pipeline delete -n %s", e.pipeline)), color.HighlightCode(fmt.Sprintf("copilot env delete -n %s", e.env)))
 }
+
+type errTaskRoleRetrievalFailed struct {
+	chainErrs []error
+}
+
+func (e *errTaskRoleRetrievalFailed) Error() string {
+	return errors.Join(e.chainErrs...).Error()
+}
+
+func (e *errTaskRoleRetrievalFailed) RecommendActions() string {
+	return fmt.Sprintf(`TaskRole retrieval failed. You can manually add permissions for your account to assume TaskRole permissions by adding the following YAML override to your service:
+%s
+For more information on YAML overrides see %s`,
+		color.HighlightCodeBlock(`- op: add
+  path: /Resources/TaskRole/Properties/AssumeRolePolicyDocument/Statement/-
+  value:
+    Effect: Allow
+    Principal:
+      AWS: "arn:aws:iam::[account-ID]:root"
```

Comment:
Would it be a good idea for us to fill this acct id for them?

Reply:
I don't like that this leaves the user in the dark of what the ID actually is 🤔

Reply:
or at least we can be clear about what account-ID they need to put in (app account or env account)